src/functions-and-arrays.js

```diff
@@ -52,14 +98,34 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
-
+function uniquifyArray(arr) {
+  if (arr.length === 0) return null;
+  let uniqueArray = [];
+  let filteredArr = arr.filter(function(item, index) {
+  if (arr.indexOf(item) == index) {
+    uniqueArray.push(arr[index])
+  }
+    });
+    return uniqueArray;   
+}
+// I tried to do this in so many different ways (with for loops and for each loops) and it didn't work; 
```

Comment:
- have you tried using `indexOf() ` as the task requires? in that case when you loop with `for ` or `forEach` you can be checking if item is already in the array 
- also there is `includes()` method that checks if item is inside an array

- We will learn filter method later this week, but well done - as long as you understand how it works!